Doc/library/typing.rst

```diff
@@ -893,8 +898,8 @@ using ``[]``.
    ``NoReturn`` can also be used as a
    `bottom type <https://en.wikipedia.org/wiki/Bottom_type>`_, a type that
    has no values. Starting in Python 3.11, the :data:`Never` type should
-   be used for this concept instead. Type checkers should treat the two
-   equivalently.
+   be used for this instead, and ``Never`` should only be used as return
+   annotation. Type checkers should treat the two equivalently.
```

Comment:
```suggestion
   be used for this instead. Type checkers should treat the two equivalently.
```
I think you mean "NoReturn" instead of "Never", but I'd prefer to keep this phrase out. That sort of thing can be used in a style guide, but I don't think the CPython documentation needs to be prescriptive here.